### PR TITLE
Fix anchor links in Make example

### PR DESCRIPTION
**Pull request by GjjvdBurg**
### Summary

Fixes broken anchor links in the Make example. There are two anchor links here so that we can link to "automatic variables" and "pattern rules" later on, but they didn't have a closing ``</a>`` tag, so the formatting spilled over (note everything is blue):

Screenshot from 2020-06-12 14-43-27

This was my mistake, so apologies it took so long to spot and fix it! :facepalm: 

### List of changes proposed in this PR (pull-request)

* Add closing tags to anchor links in Make example
* Make links to these tags in Case Study functional.

### What should a reviewer concentrate their feedback on?

- [ ] Double check that this indeed fixes the formatting issue shown in the screenshot above.
- [ ] Check that the links from Case Study to these anchors indeed work now.

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.

### CODE REVIEW

File: book/website/reproducible-research/make/make-examples.md [hunk 1]
```diff
@@ -268,7 +268,7 @@ Variables](https://www.gnu.org/software/make/manual/html_node/Automatic-Variable
 and [Pattern
 Rules](https://www.gnu.org/software/make/manual/html_node/Pattern-Rules.html#Pattern-Rules).
 
-<a name="automatic_var">
+<a class="anchor-link" name="make_automatic_var"></a>
 
 **Automatic Variables.** With automatic variables we can use the names of the
```
Comment (malvikasharan) [inline book/website/reproducible-research/make/make-examples.md:273]:
```suggestion
(rr-make-examples-automaticvar)=
#### Automatic Variables.

With automatic variables we can use the names of the
```

File: book/website/reproducible-research/make/make-examples.md [hunk 2]
```diff
@@ -305,7 +305,7 @@ the beginning (*first* prerequisite), and you can remember ``$@`` (dollar
 There are more automatic variables that you could use, see [the
 documentation](https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html).
 
-<a name="pattern_rules">
+<a class="anchor-link" name="make_pattern_rules"></a>
 
 **Pattern Rules.** Notice that the recipes for the figures have become
```
Comment (malvikasharan) [inline book/website/reproducible-research/make/make-examples.md:310]:
```suggestion
(rr-make-examples-patternrules)=
#### Pattern Rules 

Notice that the recipes for the figures have become
```

File: book/website/reproducible-research/make/make-casestudy.md [hunk 3]
```diff
@@ -57,8 +57,10 @@ of several sections:
    ```
 
    The ``%`` symbol is used to create outputs for both sources of CSV files
-   with a single rule (see [Pattern Rules](#pattern_rules)) and the rule uses
-   [automatic variables](#automatic_var) to extract the input and output
+   with a single rule (see [Pattern 
+   Rules](./make-examples.html#make_pattern_rules)) and the rule uses
+   [automatic variables](./make-examples.html#make_automatic_var) to extract 
+   the input and output
```
Comment (malvikasharan) [inline book/website/reproducible-research/make/make-casestudy.md:63]:
```suggestion
   with a single rule in {ref}`rr-make-examples-patternrules` and the rule uses in {ref}`rr-make-examples-automaticvar` to extract the input and output
```

**Comment by GjjvdBurg** [comment]
Thanks for the review and corrections @malvikasharan! I wasn't aware of the new cross-referencing system. I've accepted your suggestions and went through the chapter to make sure the other cross-references also work. I've also made a few minor changes to the text.

**Comment by malvikasharan** [comment]
Wow, thank you @GjjvdBurg 🎊 This is great :) The current cross-referencing style will allow us to cross-ref any chapter in the book independent of their location. I am happy to merge this.

**Comment by GjjvdBurg** [comment]
Great, thanks for merging @malvikasharan!

**Pull request merged**